### DISCUSSION TITLE
Launching a dev tool on Product Hunt? Drop your link here

### DISCUSSION THREAD

**Discussion by fmerian** [fmerian/awesome-product-hunt#1] [General]
So you're launching your developer tool on Product Hunt? Drop a link to your product launch here in the comments.

**Let's support developer-first products!**

> Edit: you can also ping me [@fmerian on X](https://x.com/intent/user?screen_name=fmerian) or [book a 1-hour AMA session](https://cal.link/fmerian) ✌️

**Comment by Tsabary** [comment]
Hey @fmerian I can't message you on X, but I'd love to have you as the Hunter for Replyke!

https://www.producthunt.com/products/replyke?launch=replyke-os-social-infra-layer-for-apps

**Reply by fmerian** [reply]
@Tsabary neat product! feel free to add me as a Hunter from the `/edit` page

**Reply by Tsabary** [reply]
Awesome :) :) I'll be launching this week! Thank you

EDIT: currently it says Oct 13, as I pushed it a bit hoping to connect with a quality hunter like yourself. If there's any day in which you already have a product you're hunting that launches let me know!

**Reply by fmerian** [reply]
@Tsabary may is suggest launching on Saturday, October 11?

**Reply by Tsabary** [reply]
@fmerian Mmm I think I'd like to try a work day. Should I keep it Oct 13/14 then? I'm ok with that

https://www.producthunt.com/posts/hunter-invite/BEI3O2XCYFPSOH

**Reply by fmerian** [reply]
@Tsabary Oct 13 might be the best option for you IMHO. enjoy your launch day!

**Reply by Tsabary** [reply]
@fmerian I will keep it as the original then - Oct 13 :) Thank you!

**Reply by fmerian** [reply]
rooting for you!

**Reply by fmerian** [reply]
@Tsabary congratulations on the launch! you're `#2` Product of the Day! want to create a PR and add Replyke to the list?

```
| `YYYY-MM-DD` | [Name of the product](https://www.producthunt.com/products/{yourProduct}/launches/{slug}) | Tagline | #0 | #0 | #0 |
```

**Reply by Tsabary** [reply]
Thank you @fmerian ! Yes I will :)

**Comment by rnyrms** [comment]
Hello @fmerian!

I've been looking for a Hunter on PH that has synergy with Jspreadsheet, and I stumbled upon your work. I was impressed! Nice job, man!

How do you handle Hunt on Product Hunt? We would like to know how you typically approach and if you can help us.

https://www.producthunt.com/products/jspreadsheet-2?launch=jspreadsheet-3🤘🏼

**Reply by fmerian** [reply]
@rnyrms hey Rony! you can just add me as a Hunter from the `/edit` page. no catch.

**Comment by vardhan-nahar** [comment]
Hey @fmerian 

I've been looking for a Hunter to join our launch at Turbo AI. I think you would be a great fit, let me know!

https://www.producthunt.com/products/turbo-ai-turbolearn-ai?launch=turbo-ai-turbolearn-ai

Currently I am planning to launch in the next week, let me know what day works best if you decide to join as a hunter!

**Reply by fmerian** [reply]
@vardhan-nahar thanks for reaching out! neat product. am really focus on developer tools tho, not 100% sure if i'd be the best fit for you tbh.

see [this thread](https://www.producthunt.com/p/producthunt/hunters-who-are-visible-and-active-on-product-hunt-short-list) for more suggestions. hope it helps, keep it up

**Comment by vaibs-d** [comment]
Hey @fmerian we're building an AI Data Scientist and just launched on Product Hunt: https://www.producthunt.com/products/plexe/launches/plexe-2
Would love for you try the product and give us feedback! :)

**Reply by fmerian** [reply]
@vaibs-d upvoted! neat product

**Comment by samkoesnadi** [comment]
Hi @fmerian , it is really cool what you are doing here. We are pre-launching our Computer-Using Agent tool here https://github.com/kira-id/cua.kira . It is currently in WIP, but already curious what you think about it. The goal is to eventually have an AI that can do all of our digital chores (enterprise level software work - even building games with Unity / Godot via prompt)

**Reply by fmerian** [reply]
@samkoesnadi neat product! keep us posted

**Comment by mikeheinrich** [comment]
I would appreciate feedback from this community on what I'm building. Launching today on PH
https://www.producthunt.com/products/colab-jetpack

**Comment by gauravguha** [comment]
Hi @fmerian  - love what you do for dev tools. Would you be open to helping us hunt for our Product Hunt launch for `thefrontkit.com` (Accessibility-first, token-synced frontend starter kits). I’ve got a live demo and assets ready. Happy to send details if you’re interested. thanks- gaurav

**Reply by fmerian** [reply]
@gauravguha happy to help ✌️ feel free to add me as a Hunter from the `/edit` page

**Reply by gauravguha** [reply]
Thanks a lot, @fmerian Please accept https://www.producthunt.com/posts/hunter-invite/VGNBRHDFQLSNIC

**Reply by fmerian** [reply]
@gauravguha accepted! enjoy

**Reply by fmerian** [reply]
@gauravguha I have another launch on December 1st. any chance we could reschedule it? earlier, e.g. November 30, or later, Tuesday, December 2, or Monday, December 8? Thanks

**Comment by Saga4** [comment]
Hey @fmerian We at codeflash.ai are launching on product hunt and live now: https://www.producthunt.com/products/codeflash 
We are building a general purpose code optimizer and are on a mission to make sure devs ship every commit performant.

**Reply by fmerian** [reply]
rooting for you!

**Comment by tatyhr** [comment]
Hello @fmerian, we have developed Windows tool for backing up, cloning and wiping drives: https://www.producthunt.com/products/multidrive?launch=multidrive Would you be open helping us hunt this product? Please let me know if you are interested.

**Reply by fmerian** [reply]
@tatyhr hey! working on another launch this Monday, Dec 1. any chance you could reschedule it?

**Reply by tatyhr** [reply]
@fmerian oh, I see. Would December 8th work for you?

**Reply by fmerian** [reply]
@tatyhr Dec 8 works! pro tip: nail the visual assets. see #11 and #16 for example. keep up the great work

**Reply by tatyhr** [reply]
Great, thank you!! Will do!

**Reply by tatyhr** [reply]
Please accept :) https://www.producthunt.com/posts/hunter-invite/6X3LMLIQM6E47R

**Reply by fmerian** [reply]
@tatyhr accepted! enjoy

**Comment by rahulbisht25** [comment]
Hello @fmerian!
We are launching on PH today :)
https://www.producthunt.com/products/stac

Stac is an open source Server Driven UI framework for Flutter that lets you update app interfaces instantly without rebuilds or store reviews. We are going to change how mobile app updates are shipped.

Github: https://github.com/StacDev/stac/
Website: https://stac.dev/

We were looking for Hunter to support us. Would appreciate your support <3

**Reply by fmerian** [reply]
@rahulbisht25 bravo! the product isn't on the front page tho, feels like a missed opportunity. pro tip: relaunch it. feel free to add me as a Hunter from the `/edit` page, no catch.

keep up the great work

**Comment by viveksingh-digia** [comment]
Hi @fmerian 
We'll be launching Digia on 9th Dec.
Digia is a platform to ship features to mobile apps in a Server Driven fashion much like how websites do. From complex UIs to states and animation, everything can be shipped without needing the user to update their apps.

Would be happy for your support :D
Website - www.digia.tech
Dashboard - https://app.digia.tech
PH Forum - https://www.producthunt.com/p/digia

**Reply by AnupamSingh0211** [reply]
Hi @fmerian 
Does 9th December works for you for https://www.producthunt.com/products/digia?launch=digia

**Reply by fmerian** [reply]
@viveksingh-digia @AnupamSingh0211 neat product! more into web dev tools tbh, passing on. enjoy your launch! rooting for you

**Comment by WhatAmIaTailor** [comment]
Hey @fmerian ! Thanks for sharing. We've built [Arkhet](https://www.arkhet.com), a web-based UX prototype tool for product builders, would welcome your take. Our whole approach is visual-first, so teams can mock up and text UX prototypes without prompts or code.

If this catches your interest We'll be launching on PH this Sun Dec 7 and welcome your support!

**Comment by gauravguha** [comment]
Hey @fmerian  👋
We're launching RapidNative v2 on Product Hunt soon. It's an AI app builder that generates full-stack mobile apps (database, auth, storage, and realtime) from a single prompt.
v1 launched last year and did well. v2 is a complete rebuild.
Would you be open to hunting it? Happy to give you early access.
🔗 [rapidnative.com](https://rapidnative.com/)

**Reply by fmerian** [reply]
@gauravguha neat product - happy to help

**Reply by gauravguha** [reply]
Hey @fmerian Can you accept this? https://www.producthunt.com/posts/hunter-invite/Y6MWS4YZHLSRNR

**Reply by fmerian** [reply]
@gauravguha accepted! go crush it

**Comment by Tsabary** [comment]
Hey @fmerian ! :)
Six months ago I've launched [Replyke.com](https://Replyke.com) with you as the Hunter and it made it to #2 Product of the day.

I took the last 6 months to refine it and position it better - and I'd like to launch v7 in the next couple of weeks.

Would you be open to being the hunter again? It's a dev tool of course. I would appreciate a lot the opportunity to reach the same core users with this big update.

**Reply by fmerian** [reply]
@Tsabary YESSSSS

**Reply by Tsabary** [reply]
@fmerian niceee!! Ok I'll drop a link here soon after I create the launch for you to approve:)

**Reply by Tsabary** [reply]
@fmerian scheduled to next Monday! Gives me time to work on all the assets.

https://www.producthunt.com/posts/hunter-invite/L3KAQ2HBX7DVGS

**Reply by fmerian** [reply]
@Tsabary accepted! enjoy
